src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,12 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int res;
```

Comment:
Good, but try simplifying this solution
Can we implemented this method without redundant variables (res, twin, bwin)?